packages/application-components/src/components/dialogs/internals/dialog-container.tsx

```diff
@@ -41,14 +41,14 @@ const getOverlayElement: ModalProps['overlayElement'] = (
 type Props = {
   isOpen: boolean;
   onClose?: (event: SyntheticEvent) => void;
-  size: 'm' | 'l' | 7 | 8 | 9 | 10 | 'scale';
+  size: 'm' | 'l' | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 'scale';
   zIndex?: number;
   title: string;
   children: ReactNode;
   getParentSelector: typeof getDefaultParentSelector;
 };
 const defaultProps: Pick<Props, 'size' | 'getParentSelector'> = {
-  size: 10,
+  size: 'l',
```

Comment:
We want to have a different default size per theme.
I didn't think changing the constraint 10 token to have a different values (it would have unwanted secondary effects), I used `l` here again and created a new token (`widthForDialogAsLarge`) for it which have a different value per theme.

This is a temporary measure: once the new theme styles are merged to the default theme, it could be reverted (and changed to `13`, which is the new constraint default).

Reply:
Can you leave a comment here so that we remember what to do?

Reply:
Added in this commit: 1876e41835f0bec5584510de97b26e3add5bcf97